### PR TITLE
RCAL-289: Updated Dark Exposure Keywords

### PR DESCRIPTION
**Pull request by PaulHuwe**
Updated maker utility and factory for dark ref model to include group keywords from exposure.

### CODE REVIEW

**Comment by nden** [comment]
A few files don't belong to this PR.
rad 0.9.0 was released - please update setup.cfg.

File: src/roman_datamodels/testing/factories.py [hunk 1]
```diff
@@ -383,6 +383,9 @@ def create_ref_meta(**kwargs):
         "description": _random_string("Reference description "),
         "exposure": {
             "type" : "WFI_IMAGE",
+            "ngroups" : 10,
+            "nframes" : 4,
+            "groupgap" : 7
```
Comment (ddavis-stsci) [inline src/roman_datamodels/testing/factories.py:388]: Can there be based on an actual MA table? I'd suggest one from the example tables from https://innerspace.stsci.edu/pages/viewpage.action?pageId=263655630

Comment (PaulHuwe) [reply src/roman_datamodels/testing/factories.py]: Fixed with numbers from INS.

**Review by ddavis-stsci — APPROVED** [review]
LGTM

**Pull request merged**